test-suites/helpers/make-suite.ts

```diff
@@ -476,3 +476,182 @@ export async function initializeMakeSuite() {
 
   return testEnv;
 }
+
+export async function initializeMakeSuiteForOracle() {
+  const testEnv: TestEnv = {
+    deployer: {} as SignerWithAddress,
+    poolAdmin: {} as SignerWithAddress,
+    assetListingAdmin: {} as SignerWithAddress,
+    emergencyAdmin: {} as SignerWithAddress,
+    riskAdmin: {} as SignerWithAddress,
+    gatewayAdmin: {} as SignerWithAddress,
+    users: [] as SignerWithAddress[],
+    pool: {} as IPool,
+    configurator: {} as PoolConfigurator,
+    poolDataProvider: {} as UiPoolDataProvider,
+    protocolDataProvider: {} as ProtocolDataProvider,
+    oracle: {} as PriceOracle,
+    paraspaceOracle: {} as ParaSpaceOracle,
+    weth: {} as WETH9Mocked,
+    pWETH: {} as PToken,
+    aWETH: {} as MockAToken,
+    paWETH: {} as PTokenAToken,
+    dai: {} as MintableERC20,
+    pDai: {} as PToken,
+    variableDebtDai: {} as VariableDebtToken,
+    variableDebtWeth: {} as VariableDebtToken,
+    pUsdc: {} as PToken,
+    usdc: {} as MintableERC20,
+    usdt: {} as MintableERC20,
+    nBAYC: {} as NTokenBAYC,
+    nMOONBIRD: {} as NTokenMoonBirds,
+    bayc: {} as MintableERC721,
+    addressesProvider: {} as PoolAddressesProvider,
+    registry: {} as PoolAddressesProviderRegistry,
```

Comment:
do we really need so many if we would like to have a seperate env for oracle

Reply:
I think  for now it's better to keep simple, this can be done in another pr to have multiple envs

Reply:
@GopherJ I've removed the tidy mode for simplicity, for `test_only` I would suggest we keep it, in this case we launch a live hardhat network locally and run deploy only once, then just rerun each test without deploy anything again, helps  a lot when I try to refactor oracle tests